zbus/src/lib.rs

```diff
@@ -231,10 +231,11 @@ mod tests {
     use zvariant::{OwnedObjectPath, OwnedValue, Type};
 
     use crate::{
-        blocking::{self, MessageIterator},
+        blocking::{self, proxy::ProxyImpl as _, MessageIterator},
```

Comment:
don't think we need these changes here.

Reply:
Considering your other comment, you are correct.